### ISSUE TITLE
Check user access level for slash commands

### ISSUE THREAD

**Issue by broxen** [Segs/Segs#208]
User access level is currently set to `9` (the highest) by default and is not checked when using slash commands.

We should check against when slash commands are used. We may need to check per command, as some commands can be run by regular users.

**Comment by nemerle** [comment]
Possible steps:
1. Define a 
```cpp
struct SlashCommand
{
    QStringList m_valid_prefixes;
    std::function<void(const QString &,Entity *)> m_handler;
    int m_required_access_level;
};
```
2. Convert all current slash commands to `SlashCommand`
3. Add a generic slash command handler, that will find the correct command, and check if given player has access to it.

**Comment by broxen** [comment]
I'll tackle this next, along with moving the growing list of slash commands out of MapInstance.cpp into a SlashCommands.cpp file.

I'll add #216 as it's intimately related.

Let me know if there are any other improvements that can be made here.

**Comment by broxen** [comment]
This is closed as of #264
